lib/capistrano/dkdeploy/typo3_cms.rb

```diff
@@ -27,7 +27,8 @@
       TERM: 'screen-256color'
     }
     # Path to typo3_console. Relative path to typo3_console script
-    set :path_to_typo3_console, 'bin/typo3cms'
+    set :path_to_typo3_console, File.join('bin', 'typo3cms')
+    set :path_to_typo3_cli, File.join('bin', 'typo3')
```

Comment:
Task `typo3:cms:cli:upload_wrapper` should use the new configuration option or delete task.